lib/versioning/poetry/index.spec.ts

```diff
@@ -79,21 +90,30 @@ describe('versioning/poetry/index', () => {
       ['1.9.0', true],
       ['1.9.4', true],
       ['1.9.4-beta', false],
+      ['1.9.4b0', false],
     ])('%s -> %i', (version, expected) => {
       expect(versioning.isStable(version)).toEqual(expected);
     });
   });
 
   describe('isValid(input)', () => {
-    it('should return null for irregular versions', () => {
-      expect(versioning.isValid('17.04.0')).toBeFalsy();
+    it('should support zero-padded version numbers allowed by PEP440', () => {
```

Comment:
```suggestion

    it('should support zero-padded version numbers allowed by PEP440', () => {
```

please add empty newline between new tests for readabillity